quick_start.sh

```diff
@@ -0,0 +1,159 @@
+fi
+
+kubectl_path=$(command -v kubectl || true)
+if [ -n "$kubectl_path" ]; then
+    print_partial_message " ⭐️ Found " "kubectl" ": $kubectl_path " "$BOLD"
+else
+    print_partial_message " 💥 Could not find " "kubectl" ". Please follow this link to install it..." "$BOLD"
+    print_message "" "   https://kubernetes.io/docs/tasks/tools/" "$BOLD"
+    exit 127
+fi
+
+docker_path=$(command -v docker || true)
+if [ -n "$docker_path" ]; then
+    print_partial_message " ⭐️ Found " "docker" ": $docker_path" "$BOLD"
+else
+    print_partial_message " 💥 Could not find " "docker" ". Please follow this link to install it..." "$BOLD"
+    print_message "" "   https://minikube.sigs.k8s.io/docs/drivers/docker/" "$BOLD"
+    exit 127
+fi
+
+current_user=$(whoami)
+if id -nG "$current_user" | grep -qw "docker"; then
+    print_partial_message " ⭐️ Found " "$current_user" " in the docker group" "$BOLD"
+else
+    print_partial_message " 💥 Could not find " "$current_user" " in the docker group. Please add it like this..." "$BOLD"
+    print_message "" "   sudo usermod -aG docker $current_user && newgrp docker" "$BOLD"
+    exit 1
+fi
+
+
+just_path=$(command -v just || true)
+if [ -n "$just_path" ]; then
+    print_partial_message " ⭐️ Found " "just" ": $just_path " "$BOLD"
+else
+    print_partial_message " 💥 Could not find " "just" ". Please follow this link to install it..." "$BOLD"
+    print_message "" "   https://github.com/casey/just?tab=readme-ov-file#pre-built-binaries" "$BOLD"
+    exit 127
+fi
+
+python_path=$(command -v python3 || true)
+if [ -n "$python_path" ]; then
+    print_partial_message " ⭐️ Found " "python3" ": $python_path " "$BOLD"
+else
+    print_partial_message " 💥 Could not find " "python3" ". Please follow this link to install it (or use your package manager)..." "$BOLD"
+    print_message "" "   https://www.python.org/downloads/" "$BOLD"
+    exit 127
+fi
+
+venv_status=$(python3 -m venv --help || true)
+if [ -n "$venv_status" ]; then
+    print_partial_message " ⭐️ Found " "venv" ": a python3 module" "$BOLD"
+else
+    print_partial_message " 💥 Could not find " "venv" ". Please install it using your package manager." "$BOLD"
+    exit 127
+fi
+
+bpf_status=$(grep CONFIG_BPF /boot/config-"$(uname -r)" || true)
+if [ -n "$bpf_status" ]; then
+    config_bpf=$(echo "$bpf_status" | grep CONFIG_BPF=y)
+    if [ "$config_bpf" = "CONFIG_BPF=y" ]; then
+        print_partial_message " ⭐️ Found " "BPF" ": Berkeley Packet Filters appear enabled" "$BOLD"
+    else
+        print_partial_message " 💥 Could not find " "BPF" ". Please figure out how to enable Berkeley Packet Filters in your kernel." "$BOLD"
+        exit 1
+    fi
+else
+    print_partial_message " 💥 Could not find " "BPF" ". Please figure out how to enable Berkeley Packet Filters in your kernel." "$BOLD"
+    exit 1
+fi
+
+print_message "" "" ""
+print_message "" "    Let's try to spin up a python virtual environment..." ""
+print_message "" "" ""
+
+python3 -m venv .venv
```

Comment:
Could test for an existing venv here perhaps? that said, I don't think running this script actually _overwrote_ my existing venv; more testing needed...

Reply:
Will do a check for .venv directory.